extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-api-provider.ts

```diff
@@ -10,19 +10,19 @@
 
 import { ExtPluginApiProvider, ExtPluginApi } from '@theia/plugin-ext/lib/common/plugin-ext-api-contribution';
 import { injectable } from 'inversify';
-import * as path from 'path';
 
 @injectable()
 export class ChePluginApiProvider implements ExtPluginApiProvider {
 
     provideApi(): ExtPluginApi {
+        console.log('provide che plugin api!!!');
         return {
             frontendExtApi: {
                 initPath: '/che/api/che-api-worker-provider.js',
                 initFunction: 'initializeApi',
                 initVariable: 'che_api_provider'
             },
-            backendInitPath: path.join(__dirname, '../plugin/node/che-api-node-provider.js')
+            backendInitPath: '@eclipse-che/theia-plugin-ext/lib/plugin/node/che-api-node-provider.js'
```

Comment:
in theia, we don't have path.join as well anymore ?

Reply:
I don't see no more api providers in the theia repo, only in the README.md: https://github.com/theia-ide/theia/blob/master/packages/plugin-ext/doc/how-to-add-new-plugin-namespace.md#declare-extpluginapiprovider-implementation

Reply:
it's there https://github.com/theia-ide/theia/blob/797db759bcdde66c8bbabf2ab0b041bc7d887d30/packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts#L182